x-pack/plugins/infra/public/components/logging/log_analysis_results/analyze_in_ml_button.tsx

```diff
@@ -11,10 +11,12 @@ import { encode } from 'rison-node';
 import { TimeRange } from '../../../../common/http_api/shared/time_range';
 import { useLinkProps, LinkDescriptor } from '../../../hooks/use_link_props';
 
+type DatemathRange = TimeRange | { startTime: string; endTime: string };
```

Comment:
Would it be worth limiting ourselves to one date representation for simplicity's sake?

Reply:
I'm removing this, since the time range is not needed for the job management pages